components/admin_console/billing/billing_subscriptions.tsx

```diff
@@ -5,34 +5,35 @@ import React, {useState, useEffect} from 'react';
 import {useDispatch, useStore, useSelector} from 'react-redux';
 import {FormattedMessage, useIntl} from 'react-intl';
 
-import {getCloudSubscription, getCloudProducts} from 'mattermost-redux/actions/cloud';
-import {DispatchFunc} from 'mattermost-redux/types/actions';
-
-import {PreferenceType} from 'mattermost-redux/types/preferences';
-
 import {getStandardAnalytics} from 'mattermost-redux/actions/admin';
+import {getCloudSubscription, getCloudProducts} from 'mattermost-redux/actions/cloud';
 import {savePreferences} from 'mattermost-redux/actions/preferences';
 import {getConfig, getLicense} from 'mattermost-redux/selectors/entities/general';
-import {getCurrentUser} from 'mattermost-redux/selectors/entities/users';
 import {makeGetCategory} from 'mattermost-redux/selectors/entities/preferences';
+import {getCurrentUser} from 'mattermost-redux/selectors/entities/users';
+import {DispatchFunc} from 'mattermost-redux/types/actions';
+import {PreferenceType} from 'mattermost-redux/types/preferences';
 
-import {GlobalState} from 'types/store';
-import {getCloudContactUsLink, InquiryType} from 'selectors/cloud';
-
+import {pageVisited, trackEvent} from 'actions/telemetry_actions';
 import AlertBanner from 'components/alert_banner';
 import FormattedMarkdownMessage from 'components/formatted_markdown_message';
 import FormattedAdminHeader from 'components/widgets/admin_console/formatted_admin_header';
-
+import {getCloudContactUsLink, InquiryType} from 'selectors/cloud';
+import {GlobalState} from 'types/store';
 import {Preferences, CloudBanners} from 'utils/constants';
 
 import privateCloudImage from 'images/private-cloud-image.svg';
 import upgradeMattermostCloudImage from 'images/upgrade-mattermost-cloud-image.svg';
 
-import PlanDetails from './plan_details';
 import BillingSummary from './billing_summary';
+import PlanDetails from './plan_details';
 
 import './billing_subscriptions.scss';
 
+const onUpgradeMattermostCloud = () => {
+    trackEvent('cloud_admin', 'click_upgrade_mattermost_cloud');
+};
```

Comment:
When clicking upgrade shouldn't it also open the upgrade modal?

Reply:
Yes, looks like that was missed.